### ISSUE TITLE
Database Design - Update "PurpleAir Stations"

### ISSUE THREAD

**Issue by RwHendrickson** [RwHendrickson/AQ_SpikeAlerts#33]
The database structure needs some slight adaptations for the PurpleAir Stations. This will affect the daily_updates, particularly when adding new sensors.

## Additions: 

last_elevated datetime, for use when deciding when to message people and migrating alerts, default value upon creation is... 1 day ago?

## Irrelevant Information:  

firmware_version
modified_since
uptime

# Tasks:
- update ER Diagram
- update initializedb.sql
- modify Daily_Updates.py

**Comment by RwHendrickson** [comment]
branch is here: 33-UpdatePurpleAirStations

**Comment by RwHendrickson** [comment]
Mostly done 1a76ee2 just need to change the ER Diagram

**Comment by RwHendrickson** [comment]
Incorporated a regular update of the last_elevated field in this commit e0f992c (see GetSort_Spikes.py)

**Comment by RwHendrickson** [comment]
Finished ER Diagram 781c087 - just need confirmation on e0f992c

**Comment by blackhat-hemsworth** [comment]
I didn't test this intensively, but based on what I noticed in the Modularization branch, it does seem to be working! 

I didn't realize it was done already, so I was getting confused as to why I couldn't mock up alert starts/ends just by flipping between limit of 1 vs. limit of 50 in MAIN invocation. But once I realized, I was able to adjust the time from 20 to 2 minutes, then the mocking up started actually ending old alerts & starting new ones.
